view/components/head/script.dust

```diff
@@ -10,15 +10,66 @@
 
     window.appInsights=appInsights,appInsights.queue&&0===appInsights.queue.length;
 
-    appInsights.queue.push(function() {
-      requestIDNode = document.head.querySelector("meta[name=applicationinsights-request-id]");
-      if(requestIDNode) {
-        appInsights.context.operation.id = requestIDNode.content;
+    {! If the applicationinsights-request-id meta tag is present, we add a function to the Appinsights queue which runs before our pageView and allows us to track events through the whole stack. !}
```

Comment:
The dust syntax highlighter makes comments within the script tag extremely difficult to read, would be good to add javascript comments within `{! /* */ }`